### ISSUE TITLE
Mine & Blade - Battlegear 2 Compatability

### ISSUE THREAD

**Issue by Xorbah** [Sirse/MineFantasyII-Cont#3]
Compatability with MB BG2 - Do not allow 2h weapons to be used in conjunction with Shields or other weapons? Idea.

**Comment by Sirse** [comment]
This feature are already implemented by AP in mod, but bugged. I will fix it little bit later.

**Comment by Xorbah** [comment]
Yeah, I noticed that it didn't allow me to equip 2h weapons whatsoever - it only permitted me to use it on the normal minecraft hot bar. I'm gonna start making flails, glaives and chakrams with your build of MF2 and add support for Thaumcraft, witchery and necromancy. My end goal is integration all aforementioned mods.
